<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>produits</class>
 <widget class="QMainWindow" name="produits">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1320</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>produits</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="gestion_p">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1320</width>
      <height>700</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">#gestion_p{
background-image:url(c:/Users/Dell/Downloads/background.png);
}
QTabWidget::pane{
background-image:url(c:/Users/Dell/Downloads/background.png);
}
QTabBar::tab {
  background: #6568F3;
  color: white;
  padding: 10px 12px;
border-radius:0px;
}

QTabBar::tab:selected {
    color: white;
   background-color:#FF7244;

}
QTabBar::tab:hover {
  background: #FF7244;
 }


QTabWidget::pane {
  top:0px; 
  background: rgb(245, 245, 245);
border-radius:10px;
}</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="documentMode">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab">
     <property name="styleSheet">
      <string notr="true">QTabBar::tab {
  background: #6568F3;
  color: white;
  padding: 10px 12px;
border-radius:0px;
}

QTabBar::tab:selected {
    color: white;
   background-color:#FF7244;

}
QTabBar::tab:hover {
  background: #FF7244;
 }


QTabWidget::pane {
  top:0px; 
  background: rgb(245, 245, 245);
border-radius:10px;
}</string>
     </property>
     <attribute name="title">
      <string>Gestion des produits</string>
     </attribute>
     <widget class="QGroupBox" name="bg">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1320</width>
        <height>700</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#bg{
background-image:url(c:/Users/Dell/Downloads/background.png);
}</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLineEdit" name="edit_r">
       <property name="geometry">
        <rect>
         <x>980</x>
         <y>40</y>
         <width>271</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#edit_r{

border-radius:10px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QGroupBox" name="ajouter_p">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>511</width>
         <height>271</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Montserrat ExtraBold</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#ajouter_p{
background-color:transparent;
border-radius:10px;
border: 1px solid #6568F3;
color:#6568F3;
}</string>
       </property>
       <property name="title">
        <string>FORMULAIRE</string>
       </property>
       <widget class="QLabel" name="label_n">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>50</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Montserrat ExtraBold</family>
          <pointsize>8</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Nom de produit</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_p">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>120</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Montserrat ExtraBold</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Prix unitaire</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_q">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>120</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Montserrat ExtraBold</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Quantit√©</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_t">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>50</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Montserrat ExtraBold</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Type de produit</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="edit_nom">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">#edit_nom{
border-radius:10px;
border:1px solid #6568F3;
}</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="edit_type">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>70</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">#edit_type{
border-radius:10px;
border:1px solid #6568F3;
}</string>
        </property>
       </widget>
       <widget class="QPushButton" name="valider_b">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>220</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">#valider_b{
color:white;
background-color:#6568F3;
border-radius:10px;
}
#valider_b:hover{
color:white;
background-color:#FF7244;
border-radius:10px;
}</string>
        </property>
        <property name="text">
         <string>Valider</string>
        </property>
       </widget>
       <widget class="QPushButton" name="modifier_b">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>220</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">#modifier_b{
color:white;
background-color:#6568F3;
border-radius:10px;
}
#modifier_b:hover{
color:white;
background-color:#FF7244;
border-radius:10px;
}</string>
        </property>
        <property name="text">
         <string>modifier</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="edit_qu">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>150</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">#edit_qu{
border-radius:10px;
border:1px solid #6568F3;
}</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="edit_prix">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>150</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">#edit_prix{
border-radius:10px;
border:1px solid #6568F3;
}</string>
        </property>
       </widget>
       <widget class="QTableView" name="tab_produits">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>481</width>
          <height>241</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background:transparent;
border:none;</string>
        </property>
       </widget>
       <zorder>tab_produits</zorder>
       <zorder>label_n</zorder>
       <zorder>label_p</zorder>
       <zorder>label_q</zorder>
       <zorder>label_t</zorder>
       <zorder>edit_nom</zorder>
       <zorder>edit_type</zorder>
       <zorder>valider_b</zorder>
       <zorder>modifier_b</zorder>
       <zorder>edit_qu</zorder>
       <zorder>edit_prix</zorder>
      </widget>
      <widget class="QPushButton" name="b_searsh">
       <property name="geometry">
        <rect>
         <x>1210</x>
         <y>40</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#b_searsh{
background-color:white;
border-radius:10px;
}
#b_searsh:hover{
background-color:#D6DBFD;

}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../Downloads/search.png</normaloff>../../Downloads/search.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>80</y>
         <width>931</width>
         <height>251</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="labelT1">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>-2</y>
          <width>151</width>
          <height>44</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Montserrat ExtraBold</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">#labelT1{
background-color:#D6DBFD;
color:#4A5BCF;
border-top-left-radius:15px;
}</string>
        </property>
        <property name="text">
         <string> Nom de produit</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelT1_2">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>0</y>
          <width>131</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Montserrat ExtraBold</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">#labelT1_2{
background-color:#D6DBFD;
color:#4A5BCF;

}</string>
        </property>
        <property name="text">
         <string>Type de produit</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelT1_4">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>-2</y>
          <width>111</width>
          <height>44</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Montserrat ExtraBold</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">#labelT1_4{
background-color:#D6DBFD;
color:#4A5BCF;
}</string>
        </property>
        <property name="text">
         <string>Prix unitaire</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelT1_17">
        <property name="geometry">
         <rect>
          <x>800</x>
          <y>-2</y>
          <width>101</width>
          <height>44</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Montserrat ExtraBold</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">#labelT1_17{
background-color:#D6DBFD;
color:#4A5BCF;
border-top-right-radius:15px;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="labelT1_3">
        <property name="geometry">
         <rect>
          <x>590</x>
          <y>-2</y>
          <width>101</width>
          <height>44</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Montserrat ExtraBold</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">#labelT1_3{
background-color:#D6DBFD;
color:#4A5BCF;
}</string>
        </property>
        <property name="text">
         <string> Quantit√©</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelT1_18">
        <property name="geometry">
         <rect>
          <x>690</x>
          <y>-2</y>
          <width>111</width>
          <height>44</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Montserrat ExtraBold</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">#labelT1_18{
background-color:#D6DBFD;
color:#4A5BCF;
}</string>
        </property>
        <property name="text">
         <string>   QR Code</string>
        </property>
       </widget>
       <widget class="QScrollArea" name="scrollArea">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>40</y>
          <width>721</width>
          <height>181</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:transparent;
border: 1px solid transparent;</string>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>719</width>
           <height>179</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QLabel" name="liste_p">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>40</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Montserrat ExtraBold</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#liste_p{
color:#4A5BCF;
}</string>
       </property>
       <property name="text">
        <string>Liste des produits</string>
       </property>
      </widget>
      <widget class="QFrame" name="horizontalFrame">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>350</y>
         <width>661</width>
         <height>291</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:transparent;</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout"/>
      </widget>
      <widget class="QPushButton" name="excel">
       <property name="geometry">
        <rect>
         <x>750</x>
         <y>36</y>
         <width>40</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#excel{
background-color:transparent;
border-radius:15px;
}
#excel:hover{
background-color:#FF7244;
border-radius:10px;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../Desktop/excel.png</normaloff>../../Desktop/excel.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QFrame" name="circleFrame">
       <property name="geometry">
        <rect>
         <x>-70</x>
         <y>350</y>
         <width>471</width>
         <height>311</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:transparent;</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2"/>
      </widget>
      <widget class="QPushButton" name="refreshBtn">
       <property name="geometry">
        <rect>
         <x>600</x>
         <y>10</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:transparent;
border:none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="QRCODE_3">
       <property name="geometry">
        <rect>
         <x>1110</x>
         <y>410</y>
         <width>171</width>
         <height>151</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="QRCODE_4">
       <property name="geometry">
        <rect>
         <x>1040</x>
         <y>370</y>
         <width>251</width>
         <height>211</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:10px;
border: 2px solid #6568F3;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="code_qr">
       <property name="geometry">
        <rect>
         <x>1050</x>
         <y>370</y>
         <width>251</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Montserrat ExtraBold</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#code_qr{
color:#4A5BCF;
}</string>
       </property>
       <property name="text">
        <string>               Code Qr</string>
       </property>
      </widget>
      <zorder>groupBox</zorder>
      <zorder>edit_r</zorder>
      <zorder>ajouter_p</zorder>
      <zorder>b_searsh</zorder>
      <zorder>liste_p</zorder>
      <zorder>horizontalFrame</zorder>
      <zorder>excel</zorder>
      <zorder>circleFrame</zorder>
      <zorder>refreshBtn</zorder>
      <zorder>QRCODE_4</zorder>
      <zorder>QRCODE_3</zorder>
      <zorder>code_qr</zorder>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1320</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
